cmd/plugin.go

```diff
@@ -192,6 +192,11 @@ func stashPluginMain(args []string) {
 		// Set as failure and add errors
 		resultAd.Set("TransferSuccess", false)
 		resultAd.Set("TransferError", client.GetErrors())
+		if client.ErrorsRetryable() {
+			resultAd.Set("TransferRetryable", true)
```

Comment:
In this case there was an error because we failed to run `config.InitClient` which is what the bool `isConfigErr` indicates. I just deal with the error after parsing our arguments so we can get information on the output file and such. The client should know about the error since we explicitly add the error as well:
https://github.com/PelicanPlatform/pelican/blob/e84f775368c5d981dbfb096ef7351b65de0a610c/cmd/plugin.go#L117-L125

Reply:
Ah, ok, makes sense.